main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "github.com/pyroscope-io/pyroscope/cmd"
+
+func main() {
+	cmd.Execute()
```

Comment:
The layout slightly violates [the standard](https://github.com/golang-standards/project-layout). I think we should move sub-commands to a sub-package:

```
cmd
 └──pyroscope
	├── command (or cmd)
	│	├── agent.go
	│	├── banner.go
	│	├── connect.go
	│	├── convert.go
	│	├── dbmanager.go
	│	├── server.go
	│	└── ...
	├── main.go
	├── main_unix.go
	└── main_windows.go
```

Reply:
I was thinking about this as well. On one hand - cobra provides the CLI generation tool to do it the "cobra way" but on the other hand like you said it's not fully compatible to the standard. Of course neither of these two is an official way that we have to follow. In the end I went with the cobra recommendation as we don't have the "main" entry but rather each (sub) command can be an entry to the application itself. We can of course still change this if you think it better fits how pyroscope should be organized as whole.